types/index.d.ts

```diff
@@ -1808,6 +1808,7 @@ declare namespace Knex {
     alter(): ColumnBuilder;
     queryContext(context: any): ColumnBuilder;
     withKeyName(keyName: string): ColumnBuilder;
+    after(keyName: string): ColumnBuilder;
```

Comment:
is `keyName` accurate here? I would imagine `field` or `column` would be more accurate.

Reply:
`columnName` would actually make sense, and is used slightly above in the `references` function.